packages/components/src/Enumeration/Items/Items.component.js

```diff
@@ -65,8 +65,14 @@ function Items({ items, itemsProp, currentEdit, searchCriteria }) {
 		}
 	}
 
+	function onScroll(event) {
+		if (event.target.scrollTop + event.target.clientHeight >= event.target.scrollHeight) {
```

Comment:
scrollTop, clientHeight, scrollHeight are called on every scroll event (around 400 times for 50 elements, see storybook example). These methods are time consuming. Could you "debounce" some of these calls ?